### ISSUE TITLE
jsdelivr

### ISSUE THREAD

**Issue by j2l** [saltcorn/summernote#3]
Summernote (and libraries in general) should come locally. 
Currently it's calling jsdelivr (https://cdn.jsdelivr.net/npm/summernote@0.8.18/dist/summernote-bs4.min.js) and grow its users' users' data showing just a tip of the iceberg: https://www.jsdelivr.com/statistics

Even without being used in a view, Summernote still wants to call home.

**Comment by abrain** [comment]
This also is a problem, when activating the Content Security Policy. Only local scripts are allowed (rightly so) and the summernote script from jsdelivr gets blocked.

**Comment by j2l** [comment]
Hey Andreas @abrain 
Some work has been done, but still some old plugins are calling home.
Could you please elaborate about your activation of the Content Security Policy?

**Comment by abrain** [comment]
I activated the Content Security Policy feature on the `/useradmin/http` page. After a restart, the browser console shows that the requests for https://cdn.jsdelivr.net/npm/summernote@0.8.18/dist/summernote-bs4.min.css and https://cdn.jsdelivr.net/npm/summernote@0.8.18/dist/summernote-bs4.min.js have been blocked because of the policy.

The `script-src` and `style-src` directives do not contain `https://cdn.jsdelivr.net` and force the scripts to come from the same server as the Saltcorn instance.

In the spirit of your first post I wanted to second that I'd like to see the summernote files served locally instead of pulled from a CDN.

**Comment by j2l** [comment]
> ... files served locally instead of pulled from a CDN.

I agree. I hate those "free" services (CDNs, Fonts, I see you) that are made only for analytic purpose.

Meanwhile, @abrain  you can download the js and css files, fork this plugin, and adapt the paths.
